_pages/testing.md

```diff
@@ -66,11 +66,57 @@ document:
 A YAML file can also be used to simulate an error reading or validating the document. Here is an example YAML file that does that:
 
 ```yaml
-friendly_error: |
-  We couldn't read the barcode on the back of your ID. Try taking a new picture.
-  Make sure the entire barcode is visible, clean, and doesn't have glares.
-  (Error code: 200)
+doc_auth_result: Passed #values: Passed, Failed, Attention, Caution
+image_metrics:
+  back:
+    HorizontalResolution: 300 #values: 0-600
+    VerticalResolution: 450 #values: 0-600
+    GlareMetric: 77 #values: 0-100
+    SharpnessMetric: 88 #values: 0-100
+  front:
+    HorizontalResolution: 450
+    VerticalResolution: 450
+    GlareMetric: 100
+    SharpnessMetric: 99
+failed_alerts:
+  - name: 1D Control Number Valid #See list of valid names below
+    result: Failed #values: Passed, Failed, Attention, Caution
+  - name: 2D Barcode Content
+    result: Attention
+passed_alerts:
+  - name: Visible Pattern
+    result: Passed
+liveness_result: Fail #values: Pass, Fail
```

Comment:
not super critical, but to me a space after the `#` makes comments easier to read

```suggestion
doc_auth_result: Passed # values: Passed, Failed, Attention, Caution
image_metrics:
  back:
    HorizontalResolution: 300 # values: 0-600
    VerticalResolution: 450 # values: 0-600
    GlareMetric: 77 # values: 0-100
    SharpnessMetric: 88 # values: 0-100
  front:
    HorizontalResolution: 450
    VerticalResolution: 450
    GlareMetric: 100
    SharpnessMetric: 99
failed_alerts:
  - name: 1D Control Number Valid # See list of valid names below
    result: Failed # values: Passed, Failed, Attention, Caution
  - name: 2D Barcode Content
    result: Attention
passed_alerts:
  - name: Visible Pattern
    result: Passed
liveness_result: Fail # values: Pass, Fail
```